src/PoolManager.sol

```diff
@@ -464,6 +473,12 @@ contract PoolManager is Auth, IPoolManager {
         emit RemoveVault(poolId, trancheId, asset, vault);
     }
 
+    function updateDestChainTokenPrice(uint256 price) public auth {
```

Comment:
I think this is more descriptive:
```suggestion
    function updateCentrifugeGasPrice(uint256 price) public auth {
```

Reply:
(and would update the message etc accordingly)

Reply:
I wanted to make it more generic. Not sure if it's needed thought.  If you are ok with having it specific to centrifuge, then it's fine with me